frontend/public/components/dashboard/dashboards-page/cluster-dashboard/cluster-dashboard.tsx

```diff
@@ -16,11 +17,39 @@ const mainCards = [{ Card: StatusCard }, { Card: UtilizationCard }];
 const leftCards = [{ Card: DetailsCard }, { Card: InventoryCard }];
 const rightCards = [{ Card: ActivityCard }];
 
+export const HIDE_QUICK_START_DASHBOARD_TILE_STORAGE_KEY = 'bridge/hide-quick-start-dashboard-tile';
+
 export const ClusterDashboard: React.FC<{}> = () => {
+  const isQuickStartTileHidden =
+    localStorage.getItem(HIDE_QUICK_START_DASHBOARD_TILE_STORAGE_KEY) === 'true';
   const [infrastructure, infrastructureLoaded, infrastructureError] = useK8sGet<K8sResourceKind>(
     InfrastructureModel,
     'cluster',
   );
+  const [showQuickStartCatalogTile, setShowQuickStartCatalogTile] = React.useState<boolean>(
+    !isQuickStartTileHidden,
+  );
+
+  const onQuickStartCatalogTileRemove = () => {
+    setShowQuickStartCatalogTile(false);
+  };
+
+  const rc = React.useMemo(
+    () =>
+      showQuickStartCatalogTile
```

Comment:
Why do you need this? Shouldn't the tile component not rendering anything rearrange the grid itself?

Reply:
@rohitkrai03  yes, it does but it adds a bit of space on top when I remove the quick-start tile

![Screenshot from 2020-11-10 15-59-22](https://user-images.githubusercontent.com/22490998/98673642-a6210c80-237d-11eb-9077-a1a81e7c495a.png)

Reply:
@debsmita1  Can you take a look at why it's rendering empty grid item there?